src/applications/disability-benefits/526EZ/components/FormStartControls.jsx

```diff
@@ -7,23 +7,27 @@ import SaveInProgressIntro from '../../../../platform/forms/save-in-progress/Sav
 import { ITFErrorAlert, VerifiedAlert, UnauthenticatedAlert, UnverifiedAlert, ITFDescription } from '../helpers';
 
 export default function FormStartControls(props) {
-  const { user } = props;
+  const { user: { login: { currentlyLoggedIn }, profile: { verified, services } } } = props;
 
   const somethingWentWrong = props.ITFStatus && props.ITFStatus === 'expired' || // This may not be possible
                              props.errors && props.errors.length;
+  const hasEVSSClaimsService = !!(services.find((service) => service === 'evss-claims'));
```

Comment:
Is there a list of constants that we could pull from instead of using the  `'evss-claims'` string directly?

Reply:
It looks like using the string directly is how services are typically handled, though constants are a good idea. [Ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11203) set up to discuss replacing these string instances with constants instead.

Reply:
This was actually just changed today in https://github.com/department-of-veterans-affairs/vets-website/pull/7910. Should probably happen in this PR.

Reply:
Sounds good updated.